packages/datadog-instrumentations/src/router.js

```diff
@@ -27,6 +27,9 @@ function createWrapRouterMethod (name) {
       const req = arguments[arguments.length > 3 ? 1 : 0]
       const next = arguments[lastIndex]
 
+      // explicitly call getter for express5
+      req.query
```

Comment:
In Express5, `req.query` is not parsed until it is accessed, which can delay sending it to the WAF. This might result in blocking the request too late, after headers are already sent.

To fix this, explicitly call `req.query` at the start of the request to ensure it is parsed and sent to the WAF early.